required/latex-lab/latex-lab-math.dtx

```diff
@@ -692,22 +830,22 @@
 %    \end{macrocode}
 %  a simple Desc key, take care that it is a valid string!
 %    \begin{macrocode}
-       \pdfdict_put:nne {l_pdffile/Filespec} {Desc}{(mathml-#1)} 
+       \pdfdict_put:nne {l_pdffile/Filespec} {Desc}{(mathml-#1)}
```

Comment:
general comment:  it is rather painful to read through a diff if very many changes are hust correct spacing at the end of lines. Would be really helpful to separately check in an updated version first (or at the very end) but not make those such corrections a part of a PR with "real" changes intermixed.

Reply:
I don't know if you are aware of this, but you can usually select to not show the whitespace only changes in a diff if you want to. In `git`'s command line that's e.g. `git diff -b` (or `-w` to also ignore potentially meaningful space changes), GitHub's web UI has this using the :gear: button with `Hide whitespace`

Reply:
yes, I'm aware of that and how to do it when I use diff directly, but I used the browser version of github for reviewing, e.g, starting from

https://github.com/latex3/latex2e/pull/1511/files

but ther I have no idea how to specify -b myself

Reply:
There's a menu for it at the top:
![image](https://github.com/user-attachments/assets/04868b43-ae1c-4754-bbee-579ccb2c80c5)

Reply:
ah, blind me ... I only looked at the right for config possibilities (in ... and the like)
thanks